### PR TITLE
updated solid dark button and removed borders on primary buttons

### PR DESCRIPTION
**Pull request by julio8a**
From the conversation we had Friday I:
- updated solid dark (light and dark theme)
- removed border from solid buttons

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1172,7 +1172,7 @@ <h5>Theme="dark" on blue /red (no effect on light / dark)</h5>
           <calcite-button theme="dark" title="red solid button" color='red'>red solid button</calcite-button>
 
           <calcite-button title="light solid button" color='light'>light solid button</calcite-button>
-          <calcite-button title="dark solid button" color='dark'>dark solid button</calcite-button>
+          <calcite-button theme="dark" title="dark solid button" color='dark'>dark solid button</calcite-button>
```
Comment (paulcpederson) [inline src/index.html]: so we have a `theme=dark` and a `color=dark` now?

Comment (julio8a) [reply src/index.html]: Yes, there's a light button for both light/dark theme and a dark button for both light/dark theme.

Comment (julio8a) [reply src/index.html]: Maybe `theme="dark"` should handle color too, in this case?

Comment (julio8a) [reply src/index.html]: Light theme button have `color='light'`, but no theme specified as light is the default.

Comment (paulcpederson) [reply src/index.html]: ok, no I think theme should only ever do the light/dark theme transition. We cna leave for now, maybe we could find different names in the future for the colors. As `light` and `dark` are a little abstract (more so than `red`)

Comment (julio8a) [reply src/index.html]: Sounds good

**Pull request merged**